src/applications/vaos/new-appointment/components/ContactInfoPage.jsx

```diff
@@ -91,21 +100,27 @@ const uiSchema = {
 const pageKey = 'contactInfo';
 const pageTitle = 'Your contact information';
 
-export function ContactInfoPage({
-  schema,
-  data,
-  pageChangeInProgress,
-  openFormPage,
-  updateFormData,
-  routeToNextAppointmentPage,
-  routeToPreviousAppointmentPage,
-}) {
-  const history = useHistory();
+export default function ContactInfoPage() {
   useEffect(() => {
-    openFormPage(pageKey, uiSchema, initialSchema);
     document.title = `${pageTitle} | Veterans Affairs`;
     scrollAndFocus();
   }, []);
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const pageChangeInProgress = useSelector(selectPageChangeInProgress);
+  const userData = useSelector(getFormData);
+  const email = useSelector(selectVAPEmailAddress);
+  const homePhone = useSelector(selectVAPHomePhoneString);
+  const mobilePhone = useSelector(selectVAPMobilePhoneString);
+  const { data, schema, setData } = useFormState({
+    initialSchema,
+    uiSchema,
+    initialData: {
+      ...userData,
+      email: userData.email || email,
```

Comment:
This pre-population logic was in the type of care action/reducer, but now it can be here, and it's arguably clearer.